entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md

```diff
@@ -356,6 +505,58 @@ This change was made to avoid erroneously defining a primary key property on the
 
 If the property was intended to be the foreign key, and hence part of the primary key, then explicitly configure it as such.
 
+## Database connection is now closed if not used anymore before the TransactionScope has been completed
+
+[Tracking Issue #14218](https://github.com/aspnet/EntityFrameworkCore/issues/14218)
+
+This change will be introduced in EF Core 3.0-preview 4.
+
+**Old behavior**
+
+Before EF Core 3.0, if the context was created with a closed connection it will be maintained opened while the current `TransactionScope` is active.
+
+```C#
+using (new TransactionScope())
+{
+    using (AdventureWorks context = new AdventureWorks())
+    {
+        context.ProductCategories.Add(new ProductCategory());
+        context.SaveChanges();
+
+        // Old behavior: Connection is still open at this point
+		
+        var categories = context.ProductCategories().ToList();
+    }
+}
+```
+
+**New behavior**
+
+Starting with 3.0, EF Core will close the connection as soon as it's done using it.
+
+**Why**
+
+This change was made to make the behavior more predictable and match EF6.
```

Comment:
I would like this to mention that this change was made to make certain scenarios with multiple DbContext in the same `TransactionScope` work without requiring distributed transactions.